books/free-programming-books-tr.md

```diff
@@ -142,6 +142,7 @@
 
 * [Ekonometriye Yeni Başlayanlar için Kısa bir R Kılavuzu](https://www.github.com/emraher/eybkbrk) - Emrah Er
 * [R ile Programlamaya Giriş ve Uygulamalar (2014)](http://inet-tr.org.tr/inetconf19/sunum/16.pdf) - Mustafa Gökçe Baydoğan, Berk Orbay, Uzay Çetin (PDF)
+* [R for Data Science](http://tr.r4ds.hadley.nz/) - Garrett Grolemund, Hadley Wickham
```

Comment:
- Alphabetize
- We love give also credits to traducers, so... add them to creators list using role anotations :wink:

```suggestion
* [R for Data Science](http://tr.r4ds.hadley.nz/) - Garrett Grolemund, Hadley Wickham, `trad.:` İsmail Bekar, `trad.:` Nurbahar Usta, `trad.:` Bilgecan Şen
* [R ile Programlamaya Giriş ve Uygulamalar (2014)](http://inet-tr.org.tr/inetconf19/sunum/16.pdf) - Mustafa Gökçe Baydoğan, Berk Orbay, Uzay Çetin (PDF)
```